core/src/main/java/io/questdb/std/Files.java

```diff
@@ -329,6 +329,15 @@ private static boolean strcmp(long lpsz, CharSequence s) {
 
     private static native boolean rename(long lpszOld, long lpszNew);
 
+    public static long ceilPageSize(long size) {
+        long pageCount = size / PAGE_SIZE;
```

Comment:
minor: this looks like `return (size + PAGE_SIZE - 1) / PAGE_SIZE`

Reply:
this doesn't work, produces incorrect results